### ISSUE TITLE
Terminal output contains line breaks / new lines when copying

### ISSUE THREAD

**Issue by fireattack** [microsoft/vscode#166247]
Type: <b>Bug</b>

teps to Reproduce:
 
1.  Create a python script with the content below
```
print("this is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentensethis is a very long sentense~end~")
```
2. In terminal (I use CMD, but can also be reproduced in PowerShell), run the script
3. Copy the output from VS code terminal and paste it into an empty editor tab.

Expected: the output should have no line break or new lines.

Observed: it randomly added some line breaks /  new lines. It does not happen if I use stand alone cmd window, or Windows Terminal.

![bug](https://user-images.githubusercontent.com/4130991/201563722-efd9ce73-f2b1-44b4-be5e-85eb0f0781ac.png)

I never see this happen with my other machines before. But I can reproduce this consistently with this laptop. It happens with all the outputs. I tested with stable (my profile) and insider portable (brand new profile), same issue.

VS Code version: Code - Insiders 1.74.0-insider (ef415578d1b1019db585718968a303eeca4b8979, 2022-11-11T05:19:14.666Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: unavailable_off<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|11.87GB (3.90GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30422396
pythontb:30258533
vsc_aacf:30263846
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805:30301674
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30404738
cppdebug:30492333
vsclangdf:30492506
c4g48928:30535728
dsvsc012cf:30540253
pylantcb52:30590116
pyindex848cf:30577859
nodejswelcome1:30587009
gswce1:30602633
34cci188:30599831
pyind779:30600159
dbltrim-noruby:30604474

```

</details>

**Comment by meganrogge** [comment]
This issue seems to originate from an upstream component and is not something VSCode can fix by making a change to our code. As such, I will go ahead and close it. Please try to reproduce in [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal/9n0dx20hk701?activetab=pivot:overviewtab) and report to [microsoft/terminal](https://github.com/microsoft/terminal) with repro steps.

If you cannot reproduce in Windows Terminal, chances are it's fixed in a later version of Windows. You can read more about [how we consume conpty here](https://github.com/microsoft/vscode/wiki/Terminal-Issues#why-did-you-close-my-issue). Thanks for understanding.

/label upstream terminal-conpty windows

**Comment by fireattack** [comment]
Wait what? Why does it have anything to do with Windows Terminal? I tested with CMD.exe, not WT.

And WT itself (while irrelevant) does not even have this issue.

@meganrogge : can you please clarify what you mean?

**Comment by meganrogge** [comment]
Apologies for the confusion. Here's the relevant part of my message 

If you cannot reproduce in Windows Terminal, chances are it's fixed in a later version of Windows. You can read more about how we consume conpty here. Thanks for understanding.

**Comment by std-move** [comment]
@meganrogge Any chance this could get fixed in Win10 19044/19045? The regular editions are supported at least till 2025-10-14, with IoT Enterprise LTSC (for which volume licencing is newly available) supported until 2032-01-13.

It seems like a bug that should be fixed as part of a cumulative update.

**Comment by lgarron** [comment]
> This is indeed fixed in Windows with [microsoft/terminal#6901](https://github.com/microsoft/terminal/issues/6901) when conpty started allowing the terminal to handle line wrapping. To get this change you need a later version of Windows. We hope to eventually ship conpty with Windows but we're blocked on the terminal shipping their nuget package for that.

For those of us who still see this issue on macOS, is there another issue we can follow?

**Comment by Tyriar** [comment]
@lgarron this shouldn't happen on macOS, if it does then chances are it's happening because the application you're running in the terminal moved the cursor to the next time instead of properly wrapping text.
